java/client/src/org/openqa/selenium/support/events/AbstractWebDriverEventListener.java

```diff
@@ -67,11 +67,11 @@ public void afterClickOn(WebElement element, WebDriver driver) {
     // Do nothing.
   }
 
-  public void beforeChangeValueOf(WebElement element, WebDriver driver) {
+  public void beforeChangeValueOf(WebElement element, WebDriver driver, CharSequence[] value) {
```

Comment:
change 'value' to keysToSend, here and in other references in this commit.  'value' implies the user is getting the value of the element, rather than just the keys we're sending to it.